### PR TITLE
Add usage for Event2mind

### PR DESCRIPTION
**Pull request by schmmd**
If we're able to track down commands for training and evaluation--I'd love to add those too.

### CODE REVIEW

**Comment by brendan-ai2** [comment]
The training and evaluation commands were available on the models page on the website. Why was that deleted?

See https://github.com/allenai/allennlp-website/blob/9ac12f530e90058dbc40f7754cfeb6c1ba820bab/_data/models.yml#L295.

For reference, I was able to find this deleted code using `git log -p -S event2mind`.

File: demo/package-lock.json [hunk 1]
```diff
@@ -1340,7 +1340,7 @@
     "address": {
       "version": "1.0.3",
       "resolved": "https://registry.npmjs.org/address/-/address-1.0.3.tgz",
-      "integrity": "sha512-z55ocwKBRLryBs394Sm3ushTtBeg6VAeuku7utSoSnsJKvKcnXFIyC6vh27n3rXyxSgkJBBCAvyOn7gSUcTYjg=="
+      "integrity": "sha1-tfUGMfjWzsi9IMljljr7VeBsvOk="
```
Comment (brendan-ai2) [inline demo/package-lock.json]: For my edification, what's the significance of this changing?

Comment (schmmd) [reply demo/package-lock.json]: No idea.  I don't really understand this file.  Sorry--that's a terrible answer ;-P

That file gets changed when I run the instructions for how to run the demo locally.

**Comment by schmmd** [comment]
My bad on missing that other information.  I'm merging this now but I will add what was lost in a separate PR.

**Pull request merged**